index.html

```diff
@@ -1930,6 +1945,48 @@ <h4>Representing Time</h4>
 
       </section>
 
+      <section>
+        <h3>Verifiable Credential Graphs</h3>
+      
+        <p>
+          When expressing <a>verifiable credentials</a> (for example in a <a>presentation</a>), it is
+          important to ensure that data in one <a>verifiable credential</a> is not
+          mistaken to be the same data in another <a>verifiable credential</a>. For
+          example, if one has two <a>verifiable credentials</a>, each containing an
+          object of the following form: `{"type": "Person", "name": "Jane Doe"}`, it is
+          not possible to tell if one object is describing the same person as the other
+          object. In other words, merging data between two <a>verifiable credentials</a>
+          without confirming that they are discussing the same entities and/or properties,
+          can lead to a corrupted data set.
+        </p>
+      
+        <p>
+          To ensure that data from different <a>verifiable credentials</a> are not
+          accidentally co-mingled, the concept of a
+          <dfn>verifiable credential graph</dfn> is used to
+          encapsulate each <a>verifiable credential</a>.
+          For simple <a>verifiable credentials</a>, i.e., when the JSON-LD
+          document contains a single credential with, possibly, associated proofs,
+          this graph is the <a>default graph</a>.
+          For <a>presentations</a>, each value associated with the
+          `verifiableCredential` property of the <a>presentation</a> is a separate <a>named graph</a> of type
+          <dfn class="lint-ignore">VerifiableCredentialGraph</dfn> which contains a single
+          <a>verifiable credential</a>.
+        </p>
+        <p>
+          Using these <a>graphs</a> has a concrete effect on full
+          JSON-LD processing, which properly separates graph node identifiers in one graph
```

Comment:
Whether it is "full" or not doesn't change this, and "full" is not a good name. There is static / application-specific processing that only accepts certain shapes and contexts, but that doesn't change any of the facts about the data model and this separation.

```suggestion
          Using these <a>graphs</a> has a concrete effect when performing
          JSON-LD processing, which properly separates graph node identifiers in one graph
```

Reply:
I agree (b.t.w., this text was in the spec before, I just copied it :-)